mteb/tasks/Classification/multilingual/IndicNLPNewsClassification.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification, MultilingualTask
+
+_LANGUAGES = {
+    "gu": ["guj-Gujr"],
+    "kn": ["kan-Knda"],
+    "mal": ["mal-Mlym"],
+    "mr": ["mar-Deva"],
+    "tel": ["tel-Telu"],
+    "ori": ["ori-Orya"],
+    "pa": ["pan-Guru"],
+    "ta": ["tam-Taml"],
+}
+
+
+class IndicNLPNewsClassification(MultilingualTask, AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="IndicNLPNewsClassification",
+        dataset={
+            "path": "Sakshamrzt/IndicNLP-Multilingual",
+            "revision": "3f23bd4a622a462adfb6989419cfadf7dc778f25",
+        },
+        description="A News classification dataset in multiple Indian regional languages.",
+        reference="https://github.com/AI4Bharat/indicnlp_corpus/blob/master/ai4bharat-indicnlp-corpus-2020.pdf",
```

Comment:
```suggestion
        reference="https://github.com/AI4Bharat/indicnlp_corpus#indicnlp-news-article-classification-dataset",
```
Maybe adding the llink to the dataset collection on GitHub is better